src/main/java/com/PhotomosaicSara/photomosaicImage/MainPhotomosaic.java

```diff
@@ -1,4 +1,165 @@
 package com.PhotomosaicSara.photomosaicImage;
 
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+
+import static com.PhotomosaicSara.photomosaicImage.Pixel.*;
+
 public class MainPhotomosaic {
+    private static final String TILES_DIRECTION = "source_images";
+    private static final String INPUT_IMG = "mainImage.jpg";
+    private static final String OUTPUT_IMG = "photomosaicImage.jpg";
+    private static final int TILE_WIDTH = 90;
+    private static final int TILE_HEIGHT = 90;
+    private static final int TILE_SCALE = 9;
+    private static final boolean IS_BW = false;
+
+    public static void main(String[] args) throws IOException {
```

Comment:
please create dedicated class for this business processing, check your old project and try to recreate structure, main method shouldn't contain any business logic

Reply:
Everywhere final statics, it's completely against OOP. Why didn't you compare this code with validated previous projects?